l1-contracts/test/decoders/Base.sol

```diff
@@ -98,4 +103,138 @@ contract DecoderBase is Test {
   function max(uint256 a, uint256 b) internal pure returns (uint256) {
     return a > b ? a : b;
   }
+
+  // timestamps
+
+  function assertLe(Timestamp a, Timestamp b) internal {
```

Comment:
Should this file be renamed from DecoderBase to TestBase now?

Reply:
nice error output though

Reply:
Ye probably useful. But would generally change some of the test structure. Trying out BTT for some of the gov stuff as well to see if it is useful, in that case likely makes sense to handle stuff there.